packages/fast-tooling-react/src/css-property-editor/property-editor.utilities.ts

```diff
@@ -13,11 +13,25 @@ export function getCSSPropertyConfig(data: CSSProperties): CSSPropertiesConfig {
         (dataKey: string): void => {
             dataConfig[dataKey] = {
                 value: data[dataKey],
-                keyWidth: monospaceFontWidthMultiplier * dataKey.length,
+                keyWidth:
+                    monospaceFontWidthMultiplier *
+                    filterPropertyKeyToDashSeparated(dataKey).length,
                 valueWidth: monospaceFontWidthMultiplier * data[dataKey].length,
             };
         }
     );
 
     return dataConfig;
 }
+
+export function filterPropertyKeyToDashSeparated(key: string): string {
```

Comment:
These functions seem appropriate for `@microsoft/fast-web-utilities` - `camelCaseToSnakeCase` and `snakeCaseToCamelCase`.

Reply:
perfect, I will move them.